packages/core/http/core-http-router-server-internal/src/versioned_router/oas_poc/run.js

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+/* eslint-disable no-console,@kbn/imports/no_boundary_crossing,import/no-extraneous-dependencies  */
+
+require('../../../../../../../src/setup_node_env');
```

Comment:
This wouldn't be possible if we were to implement the PoC. I assume that we'd either develop a new "version" of `config-schema` for `zod` or do a hard refactor of the existing package.